### ISSUE TITLE
Small Language Change

### ISSUE THREAD

**Issue by GlenWi** [argaar/comfortable-environment-card#12]
Are you able to change "Relatively comfort" to "Relatively comfortable"?

Many thanks.

**Comment by argaar** [comment]
You should consider the string is the "official" description presented in the original study 'Thom, E.C., "The discomfort index", Weatherwise, 12: p.57-60, 1959' and should represent the "feeling" for a human being and not the state of an environment, so you're in a state of "relatively comfort" and is not the room that is "relatively comfortable"

**Comment by GlenWi** [comment]
Hi & thanks for responding.  Whether the subject is a human being or an environment it does not make sense in the english language.  The environment or the individual can both be described as "relatively comfortable", not "relatively comfort".  The individual can experience relative comfort (he/she is relatively comfortable), or the environmental conditions support a level of relative comfort.

I did investigate your reference and it looks like Thom's original DI scale simply gives a score <21 as "No discomfort".  It doesn't seem to cater for additional categories (although I found a few that do use your terminology).  Perhaps the UTCI definitions attached are better?

In any case it's your code so your decision.  I just though it could be cleaner and more meaningful.

<img width="374" alt="Table 1" src="https://github.com/argaar/comfortable-environment-card/assets/122051976/7e0f0fcc-6bc1-455c-816c-0b5937a8bf0b">
<img width="668" alt="Another Table 1" src="https://github.com/argaar/comfortable-environment-card/assets/122051976/4438cbb0-ae3a-4746-8ffb-01bc07760383">
<img width="853" alt="UTCI" src="https://github.com/argaar/comfortable-environment-card/assets/122051976/30713210-a95d-4852-8460-653ecaf9e0d5">
